modules/ROOT/pages/building/_nav.adoc

```diff
@@ -1,5 +1,6 @@
 ** xref:building/index.adoc[Building]
 *** xref:building/creating.adoc[Creating applications and components]
+*** xref:building/imagerepository.adoc[Customizing ImageRepository]
```

Comment:
Should we move this lower so we keep "creating application and components" and "deleting application and components" next to each other

Reply:
moved

Reply:
thanks